### ISSUE TITLE
Undefined Method

### ISSUE THREAD

**Issue by mftaher** [hyperoslo/facebook-messenger#36]
getting following errors:

undefined method `app_secret=' for Facebook::Messenger::Configuration:Module (NoMethodError)

the following after commenting out app_secret
uninitialized constant Welcome (NameError)

**Comment by frodsan** [comment]
Those features haven't been released yet. You can use `master` branch:

```
gem "facebook-messenger", github: "hyperoslo/facebook-mesenger", branch: "master"
```

Or check the README for the latest version here:

https://github.com/hyperoslo/facebook-messenger/tree/7f3c6701deb730b379eb25a0c53110cfc56ab15b

@jgorset there is no release tag for 0.4.1 and 0.4.2.

**Comment by jgorset** [comment]
I pushed tags for 0.4.1, 0.4.2 and released a new version 0.5.0. Thanks guys!

**Comment by jgorset** [comment]
https://github.com/hyperoslo/facebook-messenger/blob/master/CHANGELOG.md#050---2016-05-26
